### ISSUE TITLE
Hard-coded "Your payment information is incomplete" cannot be translated.

### ISSUE THREAD

**Issue by sverleis** [Automattic/woocommerce-payments#4612]
### Description
Any user wanting to translate the field "Your payment information is incomplete", cannot do so with a translation plugin, and would have to hard-code this for every WooCommerce Payments update.

![image](https://user-images.githubusercontent.com/16252298/185940520-7121a71f-7ff4-4a5c-84de-2785596f4cd4.png)

### Acceptance criteria
Make this string translation friendly.

### Designs
None required.

### Testing instructions
1. Install any Translation extension.
1. Try and find the string under WooCommerce, WooCommerce Payments or WooCommerce Blocks.
1. Fail in your search.
1. Resort to hard-coding a solution.

### Dev notes
For reference, the core JS files here are minified in the live version of the plugin:
![image](https://user-images.githubusercontent.com/16252298/185939700-96e62e7c-b6b7-48e6-ab1a-f7a18e6c8e08.png)

### Additional context
Internal ticket `#5497343`
